vhdbuilder/lister/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Azure/agentbaker/vhdbuilder/lister
 
-go 1.21
+go 1.21.13
```

Comment:
we can actually bump this up to 1.22 at least, we were previously pinned on 1.21 since the go-dev container only had 1.21 installed